.eslintignore

```diff
@@ -4,3 +4,4 @@ assets
 cypress.json
 reporter-config.json
 dist/
+esbuild
```

Comment:
wouldn't it be better to lint the esbuild config too? eslint complains about some things like console.log() statements and imports, but i imagine it wouldn't be tough to fix or ignore in place